packages/perseus/src/components/__tests__/zoomable.test.tsx

```diff
@@ -214,8 +214,10 @@ describe("Zoomable", () => {
         // Arrange
         // Simulate window resize event
         const resizeWindowTo = (width: number, height: number) => {
-            const resizeEvent = document.createEvent("Event");
-            resizeEvent.initEvent("resize", true, true);
```

Comment:
ESLint was complaining that `initEvent` is deprecated. Given this is test code and the test still passes, I considered it a safe update.